src/agent/tools/similarity_search.py

```
@@ -0,0 +1,181 @@
+
+    **Best practices:**
+    - Requires chunk_id from previous search result
+    - Use k=5-10 for best results
+    - Set cross_document=false to search within same document only
+    - Good for discovering related content
+
+    **Method:** Dense embedding similarity (cosine)
+    
+    """
+
+    input_schema = SimilaritySearchInput
```

Ruff (0.14.5)

Undefined name `SimilaritySearchInput`

(F821)